metadata.json

```diff
@@ -18,7 +18,8 @@
         "Server 2012",
         "Server 2012 R2",
         "7",
-        "8"
+        "8",
+        "10"
```

Comment:
Just discussed with @ajjohnsonpdx - we think we should hold off on this for the moment given we don't have cells in the matrix to test.